### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by kmario23** [kmario23/deep-learning-drizzle#67] [Announcements]
## 👋
  We’ll be using Discussions as a place to connect with other members of our community, specifically for the purpose that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are interested in learning in a collaborative manner.

**Comment by AliHaider20** [comment]
I want to make a Deep learning project from start to end. Can anyone share ideas?

**Comment by rishtharamesh** [comment]
Thanks for the great post you posted. I like the way you describe the unique content. The points you raise are valid and reasonable. If any of the final year students are looking for the <a href="https://takeoffprojects.com/machine-learning-projects-for-students ">machine learning projects for students</a>

**Reply by AliHaider20** [reply]
Hi Rishtharamesh, Github is a platform to discuss ideas and collaborate to make projects. Don't spoil it by selling the projects.

**Comment by donghaiyu233** [comment]
你好，我是光学工程博士招生秘书宋岸鹏，答辩PPT已收到。

**Comment by donghaiyu233** [comment]
你好，我是光学工程博士招生秘书宋岸鹏，答辩PPT已收到。
